runner/batch.go

```diff
@@ -1,68 +0,0 @@
-package runner
```

Comment:
Should we just keep this and make batchcli support both modes since it does not seem like it would take too much. (personal issue with giving up :p )

Reply:
thinking about this more. I this we should just fork this repo to `sfn-cli` as another backend of workflow-manger. overloading the same CLI with both things seems to much at this stage.